src/read.c

```diff
@@ -3057,8 +3057,8 @@ static Int InitKernel (
 {
     TLS(ErrorLVars) = (UInt **)0;
     TLS(CurrentGlobalForLoopDepth) = 0;
-    InitGlobalBag( &ReadEvalResult, "src/read.c:ReadEvalResult" );
-    InitGlobalBag( &StackNams,      "src/read.c:StackNams"      );
+    InitGlobalBag( &TLS(ReadEvalResult), "src/read.c:ReadEvalResult" );
+    InitGlobalBag( &TLS(StackNams),      "src/read.c:StackNams"      );
```

Comment:
A lot of `InitGlobalBag` calls are modified like this... I wonder if we should hide that behind a new macro, something like this:
```
#define INIT_GLOBAL_BAG(name)   InitGlobalBag( &TLS(name), __FILE__ ":" #name )
```